Procurement/Controls/FragmentStash.xaml.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Windows.Controls;
+using Procurement.Interfaces;
+using Procurement.ViewModel.Filters;
+
+namespace Procurement.Controls
+{
+    /// <summary>
+    ///     Interaction logic for FragmentStash.xaml
+    /// </summary>
+    public partial class FragmentStash : UserControl, IStashControl
+    {
+        private readonly FragmentStashViewModel viewModel;
+        private List<IFilter> list;
+
+        public FragmentStash(int tabNumber)
+        {
+            TabNumber = tabNumber;
+
+            var stash = ApplicationState.Stash[ApplicationState.CurrentLeague].GetItemsByTab(tabNumber);
+
+            viewModel = new FragmentStashViewModel(stash);
+
+            DataContext = viewModel;
+
+            InitializeComponent();
+        }
+
+        public FragmentStash(int tabNumber, List<IFilter> list) : this(tabNumber)
+        {
+            this.list = list;
+        }
+
+        public int TabNumber { get; set; }
+        public int FilterResults { get; set; }
+        public List<IFilter> Filter { get; set; }
+
+        public void RefreshTab(string accountName)
+        {
```

Comment:
If we're not going to implement refreshing the tab in this PR, at least add a TODO to implement it later.  In the meantime, can we have some way to indicate to the user that it can not be refreshed?  Perhaps deactivate the "Refresh" item in the context menu for this tab type, or show a message box when attempting to refresh?